packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -237,7 +237,7 @@ export interface DatabaseClusterProps {
 /**
  * A new or imported clustered database.
  */
-abstract class DatabaseClusterBase extends Resource implements IDatabaseCluster {
+export abstract class DatabaseClusterBase extends Resource implements IDatabaseCluster {
```

Comment:
I wonder. Can we move this class to a different file, and export it from there, but _not_ export that file in `index.ts`? This way, it would be kept module-private, while being capable of being used in augmentations.

Reply:
I agree, one thing we've done in some of the modules is include a folder called `private` in `lib` that is never exported from `index.ts` (i.e. iam, cloudwatch, cognito, cloudfront...)

Reply:
It appears https://github.com/aws/jsii/issues/1980 is preventing me from doing that at the moment; should we "pause" this until we can do it right, or continue with this as-is?

Reply:
I would continue as-is, it might be a long time for that JSII issue to be fixed.

Reply:
agreed, we should keep on moving for now